rust/lance-datafusion/src/planner.rs

```diff
@@ -381,19 +385,15 @@ impl Planner {
             }
         }
         let context_provider = LanceContextProvider::default();
-        let mut sql_to_rel = SqlToRel::new_with_options(
+        let sql_to_rel = SqlToRel::new_with_options(
             &context_provider,
             ParserOptions {
                 parse_float_as_decimal: false,
                 enable_ident_normalization: false,
                 support_varchar_with_length: false,
+                enable_options_value_normalization: false, // TODOALEX: false or default?
```

Comment:
Please advise on what value to use here. I figured false given the other options are disabled.

https://github.com/apache/datafusion/pull/11330